test/tests/resttests.ts

```diff
@@ -237,6 +238,7 @@ describe('Rest Tests', function () {
         catch(err) {
             assert(err['statusCode'] == 500, "statusCode should be 500");
             assert(err.message && err.message.length > 0, "should have error message");
+            assert(err['responseHeaders'], "err must contain responseHeaders" + err['responseHeaders']);
```

Comment:
Will it be displayed if no `err['responseHeaders']` present?

Reply:
Thanks a lot, it shouldn't have stayed like this